subgraphs/across-v2/src/handlers/liquidity.ts

```diff
@@ -72,6 +78,20 @@ export function handleLiquidityAdded(event: LiquidityAdded): void {
   const amount = event.params.amount;
   const account = sdk.Accounts.loadAccount(event.params.liquidityProvider);
   account.liquidityDeposit(pool, amount);
+
+  // tvl
+  let inputTokenBalance: BigInt;
+  const erc20 = _ERC20.bind(event.params.l1Token);
+  const inputTokenBalanceResult = erc20.try_balanceOf(event.address);
+  if (inputTokenBalanceResult.reverted) {
+    log.info(
```

Comment:
To me this makes more sense as an error or critical log.

I am leaning towards critical bc it could throw off TVL A LOT if it reverts

Reply:
fair point. will update to critical